packages/jest-runner/src/jest-test-adapters/jest-test-adapter-factory.ts

```diff
@@ -15,7 +15,7 @@ export function jestTestAdapterFactory(
   options: StrykerOptions,
   injector: Injector<JestPluginContext>,
 ): JestGreaterThan25TestAdapter | JestLessThan25TestAdapter {
-  const version = jestWrapper.getVersion();
+  const version = semver.coerce(jestWrapper.getVersion());
   log.debug('Detected Jest version %s', version);
```

Comment:
Could we add a unit test for this? We already have a couple here: https://github.com/stryker-mutator/stryker-js/blob/renovate%2Ftypescript-5.x/packages/jest-runner/test/unit/jest-test-runner.spec.ts#L367